src/private/CABasicAnimation+MotionAnimator.h

```diff
@@ -20,9 +20,9 @@
 #import <CoreGraphics/CoreGraphics.h>
 #import <QuartzCore/QuartzCore.h>
 
-// Returns a basic animation configured with the provided timing and scale factor.
+// Returns a basic animation configured with the provided traits and scale factor.
 FOUNDATION_EXPORT
-CABasicAnimation *MDMAnimationFromTiming(MDMMotionTiming timing, CGFloat timeScaleFactor);
```

Comment:
Should this function be renamed and announced as becoming deprecated?

Reply:
Done.